tests/tests/test_mtls.py

```diff
@@ -76,7 +76,59 @@ def make_script_artifact(artifact_name, device_type, output_path):
 
 
 class TestClientMTLSEnterprise:
-    def common_test_mtls_enterprise(self, setup_ent_mtls, algorithm=None):
+    def get_key_uri_hsm(self, ssl_engine_id, device):
```

Comment:
I would separate this in two methods: `hsm_setup` (analog to `hsm_cleanup`) and `hsm_get_key_uri`

Reply:
done.